lib/treeherder.py

```diff
@@ -6,14 +6,14 @@
 from thclient import TreeherderClient
 
 
-def get_revision_hash(host, project, revision):
+def get_revision_hash(server_url, project, revision):
```

Comment:
Just to check I understand:
* there's one more usage of this function: https://github.com/mozilla/mozmill-ci/blob/bc7538268a80a254bb28f7cbd748bc30c95ef68a/lib/automation.py#L410
* but that code is currently not enabled, and if it's ever used in the future it will be updated to remove that usage?

If so, that sounds fine to me. As far as I know, taskcluster no longer supports being passed the revision hash, so that unused block doesn't work now anyway.